certifier_service/certlib/certlib_proofs.go

```diff
@@ -1685,6 +1685,24 @@ func VerifySevAttestation(serialized []byte, k *certprotos.KeyMessage) []byte {
 		return nil
 	}
 
+	vcekTcbVer := k.GetSnpTcbVersion()
+	if vcekTcbVer != ^uint64(0) {
+		tcbVer := uint64(ptr[0x180])
+		tcbVer = (uint64(ptr[0x181]) << 8) | tcbVer
+		tcbVer = (uint64(ptr[0x182]) << 16) | tcbVer
+		tcbVer = (uint64(ptr[0x183]) << 24) | tcbVer
+		tcbVer = (uint64(ptr[0x184]) << 32) | tcbVer
+		tcbVer = (uint64(ptr[0x185]) << 40) | tcbVer
+		tcbVer = (uint64(ptr[0x186]) << 48) | tcbVer
+		tcbVer = (uint64(ptr[0x187]) << 56) | tcbVer
```

Comment:
This seems like a lot of manual manipulation. Would it make more sense to use a loop here?

Reply:
Will do.